packages/world/ts/node/resolveSystems.ts

```diff
@@ -31,9 +31,10 @@ export async function resolveSystems({
   const systems = systemContracts
     .map((contract): ResolvedSystem => {
       const systemConfig =
-        config.namespaces[contract.namespaceLabel].systems[contract.systemLabel] ??
+        config.namespaces[contract.namespaceLabel]?.systems[contract.systemLabel] ??
         resolveNamespace({
           label: contract.namespaceLabel,
+          namespace: config.namespaces[contract.namespaceLabel].namespace,
```

Comment:
```suggestion
          namespace: config.namespaces[contract.namespaceLabel]?.namespace,
```
not sure if this is needed here but seems safer